test/Microsoft.AspNet.Razor.Test/Generator/CSharpTagHelperRenderingTest.cs

```diff
@@ -213,7 +213,41 @@ public static TheoryData DesignTimeTagHelperTestData
                             BuildLineMapping(1094, 30, 18, 5179, 187, 19, 29),
                             BuildLineMapping(1231, 34, 5279, 192, 0, 1),
                         }
-                    }
+                    },
```

Comment:
don't see tests for anything but bound `string` attributes.  what about unbound attributes and non-`string` bound attributes?  or am I missing something?